vm/include/lute/vm.h

```diff
@@ -16,7 +16,7 @@ namespace vm
 int lua_defer(lua_State* L);
 
 static const luaL_Reg lib[] = {
-    {"create", lua_spawn},
```

Comment:
This was actually intentional. I wanted consistency with other `create` constructors for builtins in Luau.